script/Arcueid/Day 10/10_00_ARC10_4.txt

```diff
@@ -72,17 +78,21 @@ I thought for sure this would be your first time in a place like this."
 [sha:956936902c5513fda9e448fb010ed8c981cf25c5]{
 -- Page 301, Offset 25017.
 -- 西洋の吸血鬼のイメージと、ヴローヴの服装があまりにも時代がかっていたので、勝手にゴシックな印象を抱いていた。吸血鬼社会とやらもしっかり近代化が進んでいるのかもしれない。
-The traditional western image of a vampire combined with Vlov fitting into that old-fashioned look had given me the impression that vampires are gothic creatures. Maybe even vampire society has kept up with the modern world.
+The tradition Western image of a vampire, coupled with Vlov's anachronistic clothing choices,
```

Comment:
traditional

Is "anachronistic clothing choices" really more natural than "old-fashioned look"?

Reply:
I think to _me_ I'd say yeah, at least within the structure of this sentence. 'Fitting into a look' just doesn't click.

Reply:
I can't say I agree. sounds perfectly natural to my eye's ears